### ISSUE TITLE
Example won't run and a fresh install hasn't helped

### ISSUE THREAD

**Issue by TANGOjPANDA** [jarvisteach/appJar#549]
Hello,
## Context
I was initially trying to get a graphical display working that just showed some data output but with the appjar integration of matplotlib. So a graph and some information of the side of the graph

I have tried running your example code from your website. the code is copied from the following website under the MatplotLib section, no changes have been made.
http://appjar.info/0.90/pythonDevWidgets/ 

## Any error messages produced by appJar
---
```
Traceback (most recent call last):
  File "/home/pi/Desktop/Matplot example.py", line 23, in <module>
    axes = app.addPlot("p1", *getXY())
  File "/usr/local/lib/python3.5/dist-packages/appJar/appjar.py", line 5945, in addPlot
    canvas, fig = self.__addPlotFig(title, row, column, colspan, rowspan, width, height)
  File "/usr/local/lib/python3.5/dist-packages/appJar/appjar.py", line 5969, in __addPlotFig
    canvas.show()
AttributeError: 'FigureCanvasTkAgg' object has no attribute 'show'
```

## Version Information
---
i have tried the newest version of appjar with the newest version of matplotlib
I have also tried going back a few version and on both and trying again.
I am also running it on a raspberry pi

**Comment by jarvisteach** [comment]
This has been resolved in the next release - it should be using draw()

**Comment by jarvisteach** [comment]
Resolved in issue #518
